mcp_hydrolix/mcp_env.py

```diff
@@ -168,6 +193,13 @@ def get_client_config(self) -> dict:
         if self.proxy_path:
             config["proxy_path"] = self.proxy_path
 
+        if self.service_account == True:
+            config["access_token"] = self.service_account_token
+
+        if self.service_account == False:
+            config["username"] = self.username
+            config["password"] = self.password
+
```

Comment:
nitpick: suggesting replacing two ifs with one if/else
```suggestion
          if self.service_account:
              config["access_token"] = self.service_account_token
          else:
              config["username"] = self.username
              config["password"] = self.password
  ```

Reply:
This was addressed in the latest commit.